pkg/skaffold/verify/util/logfile.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2020 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/GoogleContainerTools/skaffold/v2/pkg/skaffold/logfile"
+	"github.com/GoogleContainerTools/skaffold/v2/pkg/skaffold/output"
+)
+
+// TimeFormat is used to name log files generated by deploy step
+const TimeFormat = "2006-01-02_15-04-05"
+
+type Muted interface {
+	MuteStatusCheck() bool
+	MuteDeploy() bool
+}
+
+// WithLogFile returns a file to write the deploy output to, and a function to be executed after the deploy step is complete.
+func WithLogFile(filename string, out io.Writer, muted Muted) (io.Writer, func(), error) {
```

Comment:
I think we are not using this function anywhere (except from the tests), is this something for an upcoming change? Or is ok to remove it?

Reply:
removed now